<launch>
  <node pkg="computer_vision" type="ClassifierService.py" name="ClassifierService" output="screen" respawn="true">
  </node>
</launch>